spec/search_builders/hyrax/admin_admin_set_member_search_builder_spec.rb

```diff
@@ -11,15 +11,14 @@
                     current_user: user)
   end
   let(:user) { create(:user) }
-  let(:builder) { described_class.new(context, access) }
+  let(:builder) { described_class.new(context) }
```

Comment:
This was removed because now that I added the initialize method in the search builder, the additional parameter threw an error. Since we didn't pass anything in as the search builder was being initialized, I didn't add the parameter to my new initialize method. And adding it  would have required changing all of the collection search builder uses as well. The only time I've seen anything passed in to a search builder is when it's something other than read.

Reply:
Here is where I see this used...
https://github.com/samvera/hyrax/blob/collections-sprint/app/services/hyrax/admin_set_service.rb#L47-L50

Reply:
Also, I was the one who originally created this search builder and spec, and I started by copying some of the code from the admin_set_search_builder. That builder included access in the initialize method, and while I removed the initialize method, I missed removing that from the related spec.